tapdb/sqlc/queries/universe.sql

```diff
@@ -103,31 +103,53 @@ WHERE server_host = @target_server;
 SELECT * FROM universe_servers;
 
 -- name: InsertNewSyncEvent :exec
-WITH root_asset_id AS (
+WITH group_key_root_id AS (
+    SELECT id
+    FROM universe_roots
+    WHERE group_key = @group_key_x_only
```

Comment:
In the interest of clarity: rather than `x_only` should we use `pub` here to designate this as the public key? Or does the public key require something more by convention than what's on the x-axis?

Reply:
We always have public keys in the DB, never private keys. The difference to other public keys is that we always have 33-byte compressed public keys (that include the parity byte). But here we need to ignore that parity byte (see https://github.com/lightninglabs/taproot-assets/pull/328#issuecomment-1568794371 for some additional context) and only use the remaining 32 bytes, as mentioned here: https://github.com/lightninglabs/taproot-assets/blob/6317b3046b590cac04cd060c3c29ee947d40a07b/tapdb/sqlc/migrations/000007_universe.up.sql#L15